diff --git a/sent/Makefile b/sent/Makefile
index 731f71b..fdf942c 100644
--- a/sent/Makefile
+++ b/sent/Makefile
@@ -29,7 +29,7 @@ cscope: ${SRC} config.h
 	cscope -R -b || echo cScope not installed
 
 clean:
-	rm -f sent ${OBJ} sent-${VERSION}.tar.gz
+	rm -f sent ${OBJ} sent-${VERSION}.tar.gz config.h
 
 dist: clean
 	mkdir -p sent-${VERSION}
diff --git a/sent/config.def.h b/sent/config.def.h
index 60eb376..36385b5 100644
--- a/sent/config.def.h
+++ b/sent/config.def.h
@@ -1,16 +1,16 @@
 /* See LICENSE file for copyright and license details. */
 
 static char *fontfallbacks[] = {
-	"dejavu sans",
-	"roboto",
-	"ubuntu",
+	"Libertinus Serif",
+	"Source Serif Pro",
+	"Liberation Serif"
 };
 #define NUMFONTSCALES 42
 #define FONTSZ(x) ((int)(10.0 * powf(1.1288, (x)))) /* x in [0, NUMFONTSCALES-1] */
 
 static const char *colors[] = {
-	"#000000", /* foreground color */
-	"#FFFFFF", /* background color */
+	"#FFFFFF", /* foreground color */
+	"#000000", /* background color */
 };
 
 static const float linespacing = 1.4;
