diff --git a/config.def.h b/config.def.h
index 403daee..12b7649 100644
--- a/config.def.h
+++ b/config.def.h
@@ -10,15 +10,26 @@ static int centered = 0;                    /* -c option; centers dmenu on scree
 static int min_width = 500;                    /* minimum width when centered */
 /* -fn option overrides fonts[0]; default X11 font or font set */
 static const char *fonts[] = {
-	"monospace:size=10"
+	"monospace:size=15"
 };
-static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
+/* colorscheme */
+static const char col_alert[]    = "#f59554";
+static const char col_back[]     = "#010101";
+static const char col_four[]     = "#0097ff";
+//static const char col_back[]      = "#9997ff";
+//static const char col_back[]      = "#975fd7";
+//static const char col_back[]      = "#976f97";
+//static const char col_four[]      = "#f32323";
+
 static const char *colors[SchemeLast][2] = {
-	/*     fg         bg       */
-	[SchemeNorm] = { "#bbbbbb", "#222222" },
-	[SchemeSel] = { "#eeeeee", "#005577" },
-	[SchemeOut] = { "#000000", "#00ffff" },
+	/*               fg         bg       */
+  [SchemeNorm] = { col_four, col_back  },
+  [SchemeSel] =  { col_back, col_four  },
+  [SchemeOut] =  { col_back, col_alert },
+
 };
+
+static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 /* -l and -g options; controls number of lines and columns in grid if > 0 */
 static unsigned int lines      = 0;
 static unsigned int columns    = 0;
